src/internal/base-component/__tests__/metrics.test.ts

```diff
@@ -161,35 +159,12 @@ describe('Client Metrics support', () => {
         tryInvalidMetric('colons:not:allowed'); // invalid characters
         tryInvalidMetric('spaces not allowed'); // invalid characters
       });
-
-      describe('Metric detail validation', () => {
-        test('accepts details below the character limit', () => {
-          const validDetail = 'a'.repeat(4000);
-          metrics.sendMetric('metricName', 1, validDetail);
-          expect(window.panorama).toHaveBeenCalledWith('trackCustomEvent', {
-            eventType: 'awsui',
-            eventContext: 'metricName',
-            eventValue: '1',
-            eventDetail: validDetail,
-            timestamp: expect.any(Number),
-          });
-        });
-
-        test('throws an error when detail is too long', () => {
-          const invalidDetail = 'a'.repeat(4001);
-          metrics.sendMetric('metricName', 0, invalidDetail);
-          expect(console.error).toHaveBeenCalledWith(`Detail for metric metricName is too long: ${invalidDetail}`);
-          jest.mocked(console.error).mockReset();
-          expect(window.panorama).not.toHaveBeenCalled();
-          expect(window.AWSC.Clog.log).not.toHaveBeenCalled();
-        });
-      });
```

Comment:
Duplicate test. Removed this copy, the other one stays

https://github.com/cloudscape-design/component-toolkit/blob/e41ff4158c4c5486a42c91ecd5514846c4bf43e9/src/internal/base-component/__tests__/metrics.test.ts#L478-L489